README.md

```diff
@@ -29,14 +29,22 @@ You also need to install the dependencies required by React Native.
 Go to the [React Native environment setup](https://reactnative.dev/docs/environment-setup), then select `React Native CLI Quickstart` tab.  
 Follow instructions for your given `development OS` and `target OS`.
 
-## Quick Start
+## Getting Started
 
 - Init project
 
-```bash
-npx react-native init ProjectName --template rn-base-project
-```
-[Check out the full documentation!](#documents)
+  ```bash
+  npx react-native init YourProjectName --template rn-base-project
+  ```
+
+- Start the app
+  ```bash
+  yarn android
+  ```
+  ```bash
+  yarn ios
+  ```
+  [Check out the full documentation!](#documents)
```

Comment:
i think, we should check out full docs at folder docs 
https://github.com/saigon-technology/rn-base-project/tree/develop/docs

Reply:
I think that moving to the Document below is better than open new tab.